assets/stylesheets/modules/_conversation.scss

```diff
@@ -0,0 +1,283 @@
+      height: 126px;
+      transform: rotate(180deg);
+      width: 312px;
+    }
+  }
+
+  &.-pills {
+    .conversation__icon {
+      background-image: url('../../public/img/pills_3.png');
+      width: 80px;
+    }
+    .conversation__background {
+      background-image: url('../../public/img/section-2-illo-background.jpg');
+      height: 186px;
+      transform: rotate(180deg);
+      width: 330px;
+    }
+  }
+
+  &.-temp {
+    .conversation__icon {
+      background-image: url('../../public/img/section-3-illo.png');
+      width: 80px;
+    }
+    .conversation__background {
+      background-image: url('../../public/img/section-3-illo-background.jpg');
+      height: 166px;
+      width: 402px;
+    }
+  }
+
+  &.-box {
+    .conversation__icon{
+      top: -40px;
+      background-image: url('../../public/img/section-4-illo.png');
+      width: 260px;
+    }
+    .conversation__background {
+      background-image: url('../../public/img/section-4-illo-background.jpg');
+      height: 221px;
+      width: 558px;
+    }
+  }
+
+  &.-money {
+    margin-bottom: 140px;
+
+    .conversation__icon {
+      background-image: url('../../public/img/section-5-illo.png');
+    }
+    .conversation__background {
+      background-image: url('../../public/img/section-5-illo-background.jpg');
+      height: 187px;
+      width: 536px;
+    }
+  }
+
+  &.-cheer {
+    margin-bottom: 0px;
+
+    .conversation__icon {
+      background-image: url('../../public/img/section-6-illo.png');
+      top: -100px;
+      width: 195px;
+    }
+    .conversation__background {
+      background-image: url('../../public/img/section-6-illo-background.jpg');
+      height: 216px;
+      transform: rotate(180deg);
+      width: 396px;
+    }
+  }
+
+  .narrow-padding {
+    position: relative;
+    z-index: 1;
+  }
+}
+
+.conversation__wrap {
+  display: flex;
+  flex-direction: column;
+}
+
+.conversation__icon {
+  @extend %background-position;
+  height: 200px;
+  left: 0;
+  margin: 0 auto;
+  right: 0;
+  top: -62px;
+  width: 100px;
+  z-index: 2;
+}
+
+.conversation__background {
+  @extend %background-position;
+  left: 0;
+  margin: 0 auto;
+  right: 0;
+  top: 0;
+  z-index: 1;
+}
+
+//ANIMATE CONVERSATION
+.conversation__chat {
+  display: flex;
+  flex-direction: column;
+  margin-left: 50px;
+}
+
+//CHAT BUBBLES
+.bubble {
+  margin-bottom: 7px;
+  opacity: 0;
+  position: relative;
+  top: -10px;
+  transform-origin: left bottom;
+  transform: translateY(20px) translateX(-30px) scale(0.5);
+  transition: all 0.2s ease-out;
+
+  //BUBBLE - formatting how the bubbles look
+  p, .spinner {
+    background-color: color(lightGray);
+    border-radius: 15px;
+    color: #252224;
+    display: inline-block;
+    line-height: 1.375;
+    margin: 0;
+    padding: 12px;
+  }
+
+  //BUBBLE - typing spinner
+  .spinner {
+    bottom: 0;
+    position: absolute;
+
+    > span {
+      -webkit-animation: typing-dots 1s ease-in infinite;
```

Comment:
Do you have autoprefixer in your build tools? If not, I definitely recommend adding it.